govcd/api_vcd_test.go

```diff
@@ -131,7 +134,30 @@ func (vcd *TestVCD) SetUpSuite(test *C) {
 		panic(err)
 	}
 	// creates a new VApp for vapp tests
-	vcd.vapp = *NewVApp(&vcd.client.Client)
+	vapp, err := vcd.createTestVapp("go-vapp-tests")
+	if err != nil {
+		panic(err)
+	}
+	vcd.vapp = vapp
+}
+
+func (vcd *TestVCD) TearDownSuite(check *C) {
+	err := vcd.vapp.Refresh()
+	if err != nil {
+		panic(err)
+	}
+	task, _ := vcd.vapp.PowerOn()
```

Comment:
Why do we have a power **ON** call in a tear down app? Shouldn't it be power OFF?
Also, we should check whether the vApp exists.